django/http/request.py

```diff
@@ -402,6 +402,19 @@ def __init__(self, query_string=None, mutable=False, encoding=None):
                                 value)
         self._mutable = mutable
 
+    @classmethod
+    def fromkeys(cls, iterable, value='', mutable=False, encoding=None):
+        """
+        Returns a new QueryDict with keys from iterable and values equal to value.
+        Note: repeated keys are allowed.
+        """
+        q = cls('', mutable=True, encoding=encoding)
+        for key in iterable:
```

Comment:
It feels like we should be checking whether we've received something iterable? If an int gets passed in, what will happen?